pkg/tso/config.go

```diff
@@ -70,17 +84,60 @@ func NewConfig() *Config {
 	return &Config{}
 }
 
+const (
+	defaultLeaderLease   = int64(3)
+	defaultMaxResetTSGap = 24 * time.Hour
+
+	defaultName              = "TSO"
+	defaultBackendEndpoints  = "http://127.0.0.1:2379"
+	defaultListenAddr        = "http://127.0.0.1:2381"
+	defaultEnableGRPCGateway = true
+
+	defaultTSOSaveInterval           = time.Duration(defaultLeaderLease) * time.Second
+	defaultTSOUpdatePhysicalInterval = 50 * time.Millisecond
+	maxTSOUpdatePhysicalInterval     = 10 * time.Second
+	minTSOUpdatePhysicalInterval     = 1 * time.Millisecond
+
+	defaultDashboardAddress    = "auto"
```

Comment:
Is it necessary?

Reply:
it's deadcode. removed. will add dashboard feature in the future.